packages/@aws-cdk/alexa-ask/package.json

```diff
@@ -61,23 +61,7 @@
     "url": "https://aws.amazon.com",
     "organization": true
   },
-  "jest": {
-    "moduleFileExtensions": [
-      "js"
-    ],
-    "coverageThreshold": {
-      "global": {
-        "branches": 60,
-        "statements": 80
-      }
-    },
-    "collectCoverage": true,
-    "coverageReporters": [
-      "lcov",
-      "html",
-      "text-summary"
-    ]
-  },
+  "jest": {},
```

Comment:
Let's now make this a boolean flag, and move this under the 'cdk-build' scope.

Reply:
Done.